quint/src/ir/quintIr.ts

```diff
@@ -179,7 +179,7 @@ export const builtinOpCodes = [
   'length',
   'map',
   'mapBy',
-  'match',
+  'matchVariant',
```

Comment:
Prevents conflict with `match` keyword used in sugared `match foo { ... }` form.